### PR TITLE
Sentry

### PR DESCRIPTION
**Pull request by elena-vi**

### CODE REVIEW

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml [hunk 1]
```diff
@@ -43,6 +42,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Set up sentry cli
```
Comment (Bhavesh-Patel) [inline .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:45]: Do we want issues sent to Sentry during integration tests?
I suspect not, but if so, I'm interested in a understanding the reason.

File: tests/unit/gateways/AvailableAppointments.test.js [hunk 2]
```diff
@@ -36,22 +36,4 @@ describe('SearchProperties', () => {
     )
     expect(result).toEqual(dummyData)
   });
-
```
Comment (Bhavesh-Patel) [inline tests/unit/gateways/AvailableAppointments.test.js:39]: Can this and the other equivalent, deleted tests be replaced with new tests?

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml (hunk 1 above)
Comment (Bhavesh-Patel) [inline .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:45]: Do we want issues sent to Sentry during integration tests?
I suspect not, but if so, I'm interested in a understanding the reason.

File: api/gateways/apiRequester.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ module.exports = axios => {
 
     makePostRequest: ({uri, body ={}}) =>{
       var identifier = process.env.REPAIRS_API_IDENTIFIER
-      var baseUrl = process.env.REPAIRS_API_BASE_URL;
+      var baseUrl = process.env.REPAIRS_API_BASE_URL+'wrong';
```
[the remark this answers is not in the document]
Comment (Bhavesh-Patel) [reply api/gateways/apiRequester.js]: Already resolved.

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml (hunk 1 above)
Comment (Bhavesh-Patel) [reply .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:45]: Duplicate comment

File: pages/404.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+// export default function Custom404() {
```
[the remark this answers is not in the document]
Comment (Bhavesh-Patel) [reply pages/404.js]: Already resolved.

File: api/gateways/apiRequester.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ module.exports = axios => {
   return {
     makeGetRequest: ({uri, params ={}}) =>{
       var identifier = process.env.REPAIRS_API_IDENTIFIER
-      var baseUrl = process.env.REPAIRS_API_BASE_URL;
+      var baseUrl = process.env.REPAIRS_API_BASE_URL+'wrong';
```
[the remark this answers is not in the document]
Comment (Bhavesh-Patel) [reply api/gateways/apiRequester.js]: Already resolved.

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml (hunk 1 above)
Comment (elena-vi) [reply .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:45]: we initialise sentry, however we run in `dryRun` mode so no issues that occour during the test run will not get reported

File: tests/unit/gateways/AvailableAppointments.test.js (hunk 2 above)
Comment (elena-vi) [reply tests/unit/gateways/AvailableAppointments.test.js:39]: we are catching any exceptions thrown in the function index with sentry so these tests are now redundant - although we'd probably might want to add some tests on those files later

File: pages/_error.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import NextErrorComponent from 'next/error';
+
+import * as Sentry from '@sentry/nextjs';
+
+const MyError = ({ statusCode, hasGetInitialPropsRun, err }) => {
+  if (!hasGetInitialPropsRun && err) {
+    // getInitialProps is not called in case of
```
[the remark this answers is not in the document]
Comment (elena-vi) [reply pages/_error.js:8]: I'd be inclinded to leave them so we know what this magic generated code does

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**